src/slate-editor/slate-editor.tsx

```diff
@@ -66,7 +67,14 @@ function renderBlock(props: RenderBlockProps, editor: Editor, next: () => any) {
   return renderedBlock || next();
 }
 
-const slatePlugins = [linkPlugin, fontSizePlugin];
+const slatePlugins = [imagePlugin, linkPlugin, fontSizePlugin];
+const editorSchema = {
+  inlines: {
+    image: {
+      isVoid: true,
+    },
+  },
+};
```

Comment:
According to https://docs.slatejs.org/v/v0.47/slate-core/plugins, it is possible for plugins to make schema additions, so I think it would be better to make this schema addition there to keep the plugin better encapsulated.

Reply:
Good call.  Fixed.